### PR TITLE
Adds some logging

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
requested by @plapatin to add logging for portal guns and when mobs stop pulling other mobs

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -367,6 +367,7 @@
 /mob/living/stop_pulling()
 	if(ismob(pulling))
 		reset_pull_offsets(pulling)
+		log_game("[key_name(src)] stopped pulling [pulling] in [AREACOORD(src)]")
```
Comment (bobbahbrown) [inline code/modules/mob/living/living.dm]:
```suggestion
		src.log_message("stopped pulling [pulling] in [AREACOORD(src)]", LOG_ATTACK)
```

This should probably be in attack log as grabs & pulled from are already located there. Also use log_message to ensure admins get it on mob/user logs in-game

**Comment by Farquaar** [comment]
>tfw no lumberjack PR

File: code/modules/projectiles/guns/energy/special.dm [hunk 2]
```diff
@@ -271,6 +271,12 @@
 		return
 	p_orange.link_portal(p_blue)
 	p_blue.link_portal(p_orange)
+	log_game("[src] ([REF(src)]) linked portals between [AREACOORD(p_orange)] and [AREACOORD(p_blue)]")
+
+/obj/item/gun/energy/wormhole_projector/process_fire(atom/target, mob/living/user, message, params, zone_override, bonus_spread)
+	. = ..()
+	if(.) // it fired
+		log_game("[key_name(user)] fired a [src] ([REF(src)]) in [AREACOORD(src)]")
```
Comment (ShizCalev) [inline code/modules/projectiles/guns/energy/special.dm]: this is already logged by the projectile's `on_hit()`.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/energy/special.dm (hunk 2 above)
Comment (spookydonut) [reply code/modules/projectiles/guns/energy/special.dm]: This is more about linking the user to the specific portal gun REF()

**Comment by ninjanomnom** [comment]
Feel free to contact a maintainer about reopening this pr when you start working on it again. Or just open a new one.

**Pull request closed without merging**